diff --git a/node_modules/js-sha3/src/sha3.js b/node_modules/js-sha3/src/sha3.js
index f9a8d9c..3620d70 100644
--- a/node_modules/js-sha3/src/sha3.js
+++ b/node_modules/js-sha3/src/sha3.js
@@ -7,13 +7,13 @@
  * @license MIT
  */
 /*jslint bitwise: true */
+var WINDOW = typeof window === 'object';
+var root = WINDOW ? window : {};
 (function () {
   'use strict';
 
   var INPUT_ERROR = 'input is invalid type';
   var FINALIZE_ERROR = 'finalize already called';
-  var WINDOW = typeof window === 'object';
-  var root = WINDOW ? window : {};
   if (root.JS_SHA3_NO_WINDOW) {
     WINDOW = false;
   }
@@ -654,3 +654,5 @@
     }
   }
 })();
+
+export default root;
